wodles/azure/tests/test_azure.py

```diff
@@ -708,6 +709,47 @@ def test_get_blobs(mock_send, mock_update, blob_date, min_date, max_date, desire
             [blob.name for blob in blob_list if extension in blob.name])
 
 
+@patch('azure-logs.update_row_object')
+@patch('azure-logs.send_message')
+def test_get_blobs_only_with_prefix(mock_send, mock_update):
+    """Test get_blobs obtains the blobs only with prefix from a container and send their content to the socket."""
+    azure.args = MagicMock(
+        blobs=None, json_file=False, json_inline=False, reparse=False
+    )
+
+    prefix = "test_prefix"
+    blob_date_str = parse(FUTURE_DATE)
+
+    blob_list = [create_mocked_blob(blob_name=f"blob_{i}", last_modified=blob_date_str) for i in range(5)] + [
+        create_mocked_blob(blob_name=f"{prefix}/blob_{i}", last_modified=blob_date_str) for i in range(5)
+    ]
+
+    # The first iteration will contain a full blob list and a none next_marker
+    blob_service_iter_1 = MagicMock(next_marker=None)
+    blob_service_iter_1.__iter__ = MagicMock(return_value=iter(blob_list))
+    blob_service = MagicMock()
+    blob_service.list_blobs.return_value = blob_service_iter_1
+
+
+    with open(os.path.join(TEST_DATA_PATH, "storage_events_plain")) as f:
+        contents = f.read()
+        blob_service.get_blob_to_text.return_value = MagicMock(content=contents)
```

Comment:
There is no need for this as we are not validating the message sent to analysisd.

```suggestion
```